lldb/include/lldb/Target/RegisterFlags.h

```diff
@@ -10,6 +10,7 @@
 #define LLDB_TARGET_REGISTERFLAGS_H
 
 #include "lldb/Utility/Log.h"
+#include "lldb/Utility/StreamString.h"
```

Comment:
We can just forward declare StreamString, and possibly remove Log.h above as well if we want to.

Reply:
Done.